File: day3/day3es3.js
let somma = 0;
let conteggio = 0;
let input;

alert("Inserisci numeri. Quando hai finito, scrivi 0.");

while (true) {
  input = prompt("Inserisci un numero:");

  if (input === null || input === '0') {
    break; // Esce se l'utente annulla o inserisce 0
  }

  let numero = parseFloat(input);
  if (!isNaN(numero)) {
    somma += numero;
    conteggio++;
  }
}

if (conteggio > 0) {
  let media = somma / conteggio;
  alert(`La media è: ${media}`);
} else {
  alert("Nessun numero inserito.");
}

File: day3/day3es3.py
somma = 0
conteggio = 0

print("Inserisci numeri (0 per terminare):")

while True:
    numero = float(input("> "))
    if numero == 0:
        break
    somma += numero
    conteggio += 1

if conteggio > 0:
    media = somma / conteggio
    print(f"La media è: {media}")
else:
    print("Nessun numero inserito.")